sw_utils/execution.py

```diff
@@ -10,9 +10,8 @@
 
 def get_execution_client(endpoint: str, is_poa=False) -> Web3:
     client = Web3(
-        Web3.AsyncHTTPProvider(endpoint),
+        Web3.AsyncHTTPProvider(endpoint, request_kwargs={'timeout': 60}),
```

Comment:
May be pass it like arg with default value?
`def get_execution_client(endpoint: str, is_poa=False, timeout=60}) -> Web3:`